package/libcedarc/libcedarc.mk

```diff
@@ -0,0 +1,81 @@
+################################################################################
+#
+# libcedarc
+#
+################################################################################
+LIBCEDARC_VERSION = master
+LIBCEDARC_SITE = $(call github,aodzip,libcedarc,$(LIBCEDARC_VERSION))
```

Comment:
it's hard to say if there is any licensing on CedarX engine libs - they violated GPL and released only part of src. You could just call it PROPRIETARY, so ppl would be aware at least of situation